notebook/static/base/js/i18n.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// Module to handle i18n ( Internationalization ) and translated UI
+
+define([
+    'jed',
+    'moment',
+    'json!../../../i18n/nbjs.json',
+    'json!../../../i18n/de/LC_MESSAGES/nbjs.json',
```

Comment:
Is there a way to only load the data for languages that are needed, rather than pre-loading everything before selecting one?